pymeasure/instruments/tektronix/tek371A.py

```diff
@@ -0,0 +1,859 @@
+        dict.fromkeys(
+            [30.0, 3.0, 0.3, 0.03],
+            [1E-6, 2E-6, 5E-6, 1E-5, 20E-6, 50E-6, 100E-6, 200E-6, 500E-6, 1E-3, 2E-3]
+        )
+    )
+
+    STEP_GENERATOR_VALID_STEP_SELECTIONS_FOR_STEP_SOURCE = \
+        {
+            "VOLTAGE": STEP_GENERATOR_VALID_VOLTAGE_STEP_SELECTIONS_VS_PEAKPOWER,
+            "CURRENT": STEP_GENERATOR_VALID_CURRENT_STEP_SELECTIONS_VS_PEAKPOWER
+        }
+
+    STEP_GENERATOR_NUMBER_OF_STEPS_RANGE = [0, 5]
+
+    HORIZONTAL_DISPLAY_SENSITIVITY_VALID_SELECTIONS_VS_PEAKPOWER_FOR_COLLECTOR_SOURCE = \
+        {
+            3000.0: [100E-3, 200E-3, 0.5, 1, 2, 5],
+            300.0: [100E-3, 200E-3, 0.5, 1, 2, 5],
+            30.0: [50, 100, 200, 500],
+            3.0: [50, 100, 200, 500]
+        }
+
+    HORIZONTAL_DISPLAY_SENSITIVITY_VALID_SELECTIONS_VS_PEAKPOWER_FOR_STEPGEN_SOURCE = \
+        dict.fromkeys(
+            [3000, 300, 30, 3], [100E-3, 200E-3, 0.5, 1, 2, 5]
+        )
+
+    HORIZONTAL_DISPLAY_SENSITIVITY_VALID_SELECTIONS_VS_PEAKPOWER_FOR_SOURCE = \
+        dict.fromkeys(
+            ["STP", "STPgen", "STPGEN"],
+            HORIZONTAL_DISPLAY_SENSITIVITY_VALID_SELECTIONS_VS_PEAKPOWER_FOR_STEPGEN_SOURCE
+        )
+
+    HORIZONTAL_DISPLAY_SENSITIVITY_VALID_SELECTIONS_VS_PEAKPOWER_FOR_SOURCE.update(
+        dict.fromkeys(
+            ["COL", "COLlect", "COLLECT"],
+            HORIZONTAL_DISPLAY_SENSITIVITY_VALID_SELECTIONS_VS_PEAKPOWER_FOR_COLLECTOR_SOURCE
+        )
+    )
+
+    VERTICAL_DISPLAY_SENSITIVITY_VALID_SELECTIONS_VS_PEAKPOWER_FOR_COLLECTOR_SOURCE = \
+        {
+            3000.0: [1, 2, 5, 10, 20, 50],
+            300.0: [0.5, 1, 2, 5],
+            30.0: [100E-6, 200E-6, 500E-6, 1E-3, 2E-3, 5E-3],
+            3.0: [10E-6, 20E-6, 50E-6, 100E-6, 200E-6, 500E-6]
+        }
+
+    VERTICAL_DISPLAY_SENSITIVITY_VALID_SELECTIONS_VS_PEAKPOWER_FOR_SOURCE = \
+        dict.fromkeys(
+            ["COL", "COLlect", "COLLECT"],
+            VERTICAL_DISPLAY_SENSITIVITY_VALID_SELECTIONS_VS_PEAKPOWER_FOR_COLLECTOR_SOURCE
+        )
+
+    def __init__(self,
+                 adapter,
+                 query_delay=0.1,
+                 write_delay=0.4,
+                 timeout=5000,
+                 **kwargs):
+        super().__init__(
+            adapter,
+            "Tektronix Curve Tracer model 371A",
+            write_termination="\n",
+            read_termination="",
+            send_end=True,
+            includeSCPI=False,
+            timeout=timeout,
+            **kwargs
+        )
+
+        self.write_delay = write_delay
+        self.query_delay = query_delay
+        self.last_write_timestamp = 0.0
+        self.srq_called = False
+
+    # ############################################
+    # # Tektronix Curve Tracer 371A SCPI commands.
+    # ############################################
+
+    id = Instrument.measurement(
+        "ID?",
+        """Return the identification of the instrument (string) """,
```

Comment:
For a `measurement`, the docstring should start with `"Measure"` or `"Set"`, not with "Return".
There is a test in the current master, which will fail, unless you comply with this norm.

Reply:
Thanks Benedikt